src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -78,31 +78,23 @@ Truffle requires you to have a Migrations contract in order to use the Migration
 Filename: `contracts/Migrations.sol`
 
 ```solidity
-pragma solidity ^0.4.8;
+pragma solidity >=0.4.22 <0.9.0;
 
 contract Migrations {
-  address public owner;
-
-  // A function with the signature `last_completed_migration()`, returning a uint, is required.
```

Comment:
Could you put this comment back in please?

Reply:
done